test/devapps/authorization_code_sample.go

```diff
@@ -53,12 +53,19 @@ func getToken(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, "Access token is "+result.GetAccessToken())
 }
 
+// TODO(msal expert): This test doesn't do anything that I can tell. We call msal.NewPublicClientApplication(), but we
+// don't use it.  Someone needs to clarify to me how this is supposed to work.
```

Comment:
yeah, this is on my list of TODOs. This sample didn't make sense to me, but I see someone has [tested it and checked it off](https://github.com/AzureAD/microsoft-authentication-library-for-go/projects/2#card-47369448), so idk. will try to take a look this weekend.